build.sbt

```diff
@@ -15,7 +15,7 @@ ThisBuild / licenses := Seq("MIT" -> url("https://opensource.org/licenses/MIT"))
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
 ThisBuild / developers := List(
-  Developer(id = "jeffmay", name = "Jeff May", email = "jeff.n.may@gmail.com", url = url("https://github.com/jeffmay")),
+  Developer(id = "jeffmay", name = "Jeff May", email = "jeff.n.may@gmail.com", url = url("https://github.com/jeffmay"))
```

Comment:
You sure you don't want to add your name here? 😄 

Also, these trailing commas are inserted by my auto-formatter because I like to avoid git conflicts. I will probably revert these changes (either intentionally or accidentally) in the future.

Reply:
haha, thanks for the vote of confidence!